Source/WebCore/css/StyleProperties.cpp

```diff
@@ -593,12 +543,8 @@ String StyleProperties::fontVariantValue(const StylePropertyShorthand& shorthand
     auto ligaturesKeyword = propertyAsValueID(CSSPropertyFontVariantLigatures);
 
     for (auto property : shorthand) {
-        int foundPropertyIndex = findPropertyIndex(property);
-        if (foundPropertyIndex == -1)
-            return emptyString(); // All longhands must have at least implicit values.
-
         // Bypass getPropertyCSSValue which will return an empty string for system keywords.
-        auto value = propertyAt(foundPropertyIndex).value();
+        auto value = propertyAt(findPropertyIndex(property)).value();
```

Comment:
I think this could just use `getPropertyCSSValue`. You would need to check for null, but it would replace the `isSystemFontShorthand` check below.

Reply:
Good idea. That would be better.

Reply:
Done.